src/components/dropdown-menu/styled.ts

```diff
@@ -0,0 +1,108 @@
+import styled, { css, createGlobalStyle } from 'styled-components';
+import { getConfig } from '../../config';
+
+const isTransportportal = getConfig().themeNap;
+
+const onMobileView = '@media (max-width: 990px)';
+
+type globalProps = {
+  dropdownOpen: boolean;
+};
+
+const GlobalStyle = createGlobalStyle<globalProps>`
+  body {
+    ${onMobileView} {
+      ${({ dropdownOpen }) =>
+        dropdownOpen &&
+        css`
+          height: 100vh;
+          overflow-y: hidden;
+
+          // For safari:
+          position: fixed;
+          left: 0;
+          right: 0;
+        `}
+    }
+  }
+`;
+
+type dropdownMenuProps = {
+  desktopView: boolean;
+  mobileView: boolean;
+};
+
+const DropdownMenu = styled.nav<dropdownMenuProps>`
+  color: ${({ theme }) => theme.extendedColors.neutralDarkest};
+  display: ${({ desktopView }) => (desktopView ? 'inherit' : 'none')};
+
+  ${onMobileView} {
+    display: ${({ mobileView }) => (mobileView ? 'inherit' : 'none')};
+  }
+`;
+
+type titleProps = {
+  caret: boolean;
+};
+
+const Title = styled.button<titleProps>`
+  padding: 10px;
+  font-size: 16px;
+  border: none;
+  ${() =>
```

Comment:
Det er sikkert mulig å få tak i theme her

Reply:
Var originalt definert vha. theme, men de funket ikke på NAP sin portal. Bakgrunnen på menyknappen ble en annen farge enn headeren da NAP bruker "header" og FDK bruker "headerBg". Samme gjaldt for dropdownen, og måtte ha lys menyknapptekst ettersom NAP sin bakgrunn er hvit